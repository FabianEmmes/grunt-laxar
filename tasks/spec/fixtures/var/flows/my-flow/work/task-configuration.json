{
   "laxar-artifacts": {
      "my-flow": {
         "src": "application/flow/flow.json",
         "dest": "var/flows"
      }
   },
   "laxar-build-flow": {
      "options": {
         "userTasks": []
      },
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            }
         }
      }
   },
   "laxar-resources": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            }
         }
      }
   },
   "laxar-dependencies": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            }
         }
      }
   },
   "laxar-configure-watch": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            }
         }
      }
   },
   "laxar-dist-flow": {
      "options": {
         "userTasks": []
      },
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            }
         }
      }
   },
   "laxar-dist-css": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            }
         }
      }
   },
   "laxar-dist-js": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            }
         }
      }
   },
   "laxar-test-flow": {
      "options": {
         "userTasks": []
      },
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            }
         }
      }
   },
   "laxar-test-configure-widget": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            }
         }
      }
   }
}
