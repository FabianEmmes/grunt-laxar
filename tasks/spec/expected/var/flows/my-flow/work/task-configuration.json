{
   "laxar-artifacts": {
      "my-flow": {
         "src": "application/flow/flow.json",
         "dest": "var/flows"
      }
   },
   "laxar-build-flow": {
      "options": {
         "userTasks": []
      },
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            },
            "testDirectory": "var/tests"
         }
      }
   },
   "laxar-merge-require-config": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            },
            "testDirectory": "var/tests"
         }
      }
   },
   "laxar-resources": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            },
            "testDirectory": "var/tests"
         }
      }
   },
   "laxar-dependencies": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            },
            "testDirectory": "var/tests"
         }
      }
   },
   "laxar-configure-watch": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            },
            "testDirectory": "var/tests"
         }
      }
   },
   "laxar-dist-flow": {
      "options": {
         "userTasks": []
      },
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            },
            "testDirectory": "var/tests"
         }
      }
   },
   "laxar-dist-css": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            },
            "testDirectory": "var/tests"
         }
      }
   },
   "laxar-dist-js": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            },
            "testDirectory": "var/tests"
         }
      }
   },
   "laxar-test-flow": {
      "options": {
         "userTasks": []
      },
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            },
            "testDirectory": "var/tests"
         }
      }
   },
   "laxar-test-configure-flow": {
      "my-flow": {
         "src": "var/flows",
         "options": {
            "flow": {
               "target": "my-flow",
               "src": "application/flow/flow.json"
            },
            "testDirectory": "var/tests"
         }
      }
   }
}
